### PR TITLE
Allow configuration of Instance Divisor

### PR DESCRIPTION
**Pull request by benaadams**
Never thought was ever a need for this to be anything other than one; but you learn something everyday 😄

### CODE REVIEW

**Comment by deltakosh** [comment]
Me neither to be honest:)

**Comment by benaadams** [comment]
Half a mesh; axis multiplier -1, 1

**Comment by benaadams** [comment]
Use half a mesh; reflect with axis multiplier -1, 1

**Comment by benaadams** [comment]
Alas it doesn't repeat when it gets to end of 1 multipler; so either double draw or double instance buffer ;(

**Comment by deltakosh** [comment]
Hey..not sure to get what is the problem :(

**Comment by benaadams** [comment]
```
[v,v,v,v,v,v,v,v,v][v,v,v,v,v,v,v,v,v][v,v,v,v,v,v,v,v,v][v,v,v,v,v,v,v,v,v] // repeat divisor 0
[       pos,               pos       ][       pos,               pos       ] // repeat divisor 1 
[                 adj,                                 adj                 ] // divisor 2
```
Was hoping divisor 2 would cause divisor 1 to repeat when it got to the end. But instead just run out of vertexes :)

Was a misunderstanding on how it worked

**Comment by deltakosh** [comment]
Oh..ok :) part of me was stressed that we had a bug since long time :D

**Comment by benaadams** [comment]
tl;dr multiplier > 1 can be useful; but way way less useful than I was hoping (borderline not useful)
